bin/grpc/servers/podstore/server.go

```diff
@@ -29,9 +29,9 @@ func main() {
 	// Parse custom flags + standard Consul routing options
 	_, opts, _ := flags.ParseWithConsulOptions()
 
-	client := kp.NewConsulClient(opts)
-	podStore := kp_podstore.NewConsul(client.KV())
-	podStatusStore := podstatus.NewConsul(statusstore.NewConsul(client), kp.PreparerPodStatusNamespace)
+	client := consul.NewConsulClient(opts)
+	podStore := consul_podstore.NewConsul(client.KV())
+	podStatusStore := podstatus.NewConsul(statusstore.NewConsul(client), consul.PreparerPodStatusNamespace)
```

Comment:
pod status namespaces will likely carry over to other datastores as well, but I don't know where else to put it at the moment